analyzer/runtime/extract.go

```diff
@@ -344,7 +342,7 @@ func ExtractRound(blockHeader nodeapi.RuntimeBlockHeader, txrs []nodeapi.Runtime
 					}
 					// Set the 'Success' field to 'Pending' for deposits. This is because the outcome of the Deposit tx is only known in the next block.
 					blockTransactionData.Success = nil
-
+					blockTransactionData.RelatedAccountAddresses[to] = struct{}{}
```

Comment:
oh I see. so here for example there's additional logic used in arriving at the "to" address that we want to apply before adding the "to" address to the related accounts

although we lucked out here because the signers are elsewhere added to the related accounts set